static/app/components/datePageFilter.tsx

```diff
@@ -116,4 +121,4 @@ const DropdownTitle = styled('div')`
   flex: 1;
 `;
 
-export default withRouter(DatePageFilter);
+export default withRouter<Props>(DatePageFilter);
```

Comment:
oh woops I recently fixed this because I was using withRouter incorrectly, once you rebase off master you won't need this guy anymore

Reply:
oh that's good to know! - removed it when resolving the merge conflicts w/ master.